src/types/clickEvents.ts

```diff
@@ -0,0 +1,3 @@
+export interface RelatedSearchClick {
+    preventDefault: () => void; target: { href: any; textContent: any }
```

Comment:
better to use `href?: string; textContent?: string` if you know the value is either string or undefined, again minimise the use of `any`